### PR TITLE
BLADE-409 support migration of portal jars in liferay-plugin-package.…

### PR DESCRIPTION
**Pull request by jtydhr88**
…properties

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: cli/src/main/resources/62portaldependencyjars.properties [hunk 1]
```diff
@@ -0,0 +1,238 @@
+httpmime.jar=org.apache.httpcomponents,httpmime,4.5
```
Comment (gamerson) [inline cli/src/main/resources/62portaldependencyjars.properties]: Can you rename this to portal-dependency-jars-62.properties?

File: cli/src/main/resources/62portaldependencyjars.properties [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+isoparser.jar=com.googlecode.mp4parser,isoparser,1.0-RC-1
```
Comment (gamerson) [inline cli/src/main/resources/62portaldependencyjars.properties]: Can you sort this file?

File: cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java [hunk 3]
```diff
@@ -570,6 +573,50 @@ private void _convertToWarProject(File warsDir, File pluginDir) {
 				ivyFile.delete();
 			}
 
+			File liferayPluginPackageFile =
```
Comment (gamerson) [inline cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java]: Can you add unit tests for this behavior?

Comment (jtydhr88) [reply cli/src/main/java/com/liferay/blade/cli/command/ConvertCommand.java]: will do soon

File: cli/src/main/resources/62portaldependencyjars.properties (hunk 2 above)
Comment (jtydhr88) [reply cli/src/main/resources/62portaldependencyjars.properties]: will do soon

**Pull request closed without merging**